src/cgame/cg_view.cpp

```diff
@@ -222,6 +222,48 @@ static void CG_AddTestModel()
 	}
 }
 
+// Textures may be subject to the map's light grid
+// Some textures don't work: deformvertexes autosprite/autosprite2, sky...
+class TestShaderCmd : public Cmd::StaticCmd
+{
+public:
+	TestShaderCmd() : StaticCmd("testshader", "draw a polygon with specified q3shader") {}
+
+	void Run( const Cmd::Args &args ) const override
+	{
+		if ( args.Argc() < 2 )
+		{
+			cg.hTestShader = 0;
```

Comment:
Print a message that it's disabling so people running this command without any arguments and expecting usage are less confused?